_aggregations/metric/median-absolute-deviation.md

```diff
@@ -9,70 +9,37 @@ redirect_from:
 
 # Median absolute deviation aggregations
 
-The `median_absolute_deviation` metric is a single-value metric aggregation that returns a median absolute deviation field. Median absolute deviation is a statistical measure of data variability. Because the median absolute deviation measures dispersion from the median, it provides a more robust measure of variability that is less affected by outliers in a dataset. 
+The `median_absolute_deviation` metric is a single-value metric. Median absolute deviation is a variability metric that measures dispersion from the median.
 
-Median absolute deviation is calculated as follows:<br>
-median_absolute_deviation = median(|X<sub>i</sub> - Median(X<sub>i</sub>)|)
+Median absolute deviation is less affected by outliers than standard deviation, which relies on squared error terms, and is useful for describing data that is not normally distributed.
 
-The following example calculates the median absolute deviation of the `DistanceMiles` field in the sample dataset `opensearch_dashboards_sample_data_flights`:
+Median absolute deviation is computed as follows:<br/>
+median_absolute_deviation = median( | X<sub>i</sub> - Median(X<sub>i</sub>) | )
 
+In practice, for large datasets `median_absolute_deviation` is estimated. You can adjust the trade-off between estimation accuracy and performance; see [Adjusting estimation accuracy](#adjusting-estimation-accuracy).
 
-```json
-GET opensearch_dashboards_sample_data_flights/_search
-{
-  "size": 0,
-  "aggs": {
-    "median_absolute_deviation_DistanceMiles": {
-      "median_absolute_deviation": {
-        "field": "DistanceMiles"
-      }
-    }
-  }
-}
-```
-{% include copy-curl.html %}
+## Parameters
 
-#### Example response
+The `median_absolute_deviation` aggregation takes the following parameters.
 
-```json
-{
-  "took": 35,
-  "timed_out": false,
-  "_shards": {
-    "total": 1,
-    "successful": 1,
-    "skipped": 0,
-    "failed": 0
-  },
-  "hits": {
-    "total": {
-      "value": 10000,
-      "relation": "gte"
-    },
-    "max_score": null,
-    "hits": []
-  },
-  "aggregations": {
-    "median_absolute_deviation_distanceMiles": {
-      "value": 1829.8993624441966
-    }
-  }
-}
-```
+| Parameter | Required/Optional | Data type      | Description |
+| :--       | :--               | :--            | :--         |
+| `field`   | Required          | String         | The name of the numeric field for which the median absolute deviation is computed. |
+| `missing` | Optional          | Numeric        | The value to assign missing instances of the field. If not provided, documents with missing values are omitted from the average. |
```

Comment:
```suggestion
| `missing` | Optional          | Numeric        | The value to assign missing instances of the field. If not provided, documents with missing values are omitted from the deviation calculation. |
```

Reply:
You can just say `calculation`, else say "`median_absolute_deviation` calculation" to be more precise. I prefer the former as we are just talking about `median_absolute_deviation` in this entire page.

Reply:
```suggestion
| `missing` | Optional          | Numeric        | The value to assign missing instances of the field. If not provided, documents with missing values are omitted from the calculation. |
```

Reply:
Sorry for the back and forth, let's call it `estimation` just to be correct.

Reply:
```suggestion
| `missing` | Optional          | Numeric        | The value to assign missing instances of the field. If not provided, documents with missing values are omitted from the estimation. |
```